### PR TITLE
Clean up unused codes and dependencies

### PR DESCRIPTION
**Pull request by sima-zhu**
The PR is clean up unused docs script, unused GithubNameResolver class and unused kusto dependencies.

Passed the pipeline. Search the deleted one in tool and lang repo. No usage based on the search

### CODE REVIEW

File: tools/notification-configuration/github-codeowner-subscriber/Program.cs [hunk 1]
```diff
@@ -47,14 +47,18 @@ class Program
...
                     loggerFactory.CreateLogger<AzureDevOpsService>()
                 );
                 
                 var gitHubServiceLogger = loggerFactory.CreateLogger<GitHubService>();
                 var gitHubService = new GitHubService(gitHubServiceLogger);
-                var credential = new ClientSecretCredential(aadTenantVar, aadAppIdVar, aadAppSecretVar);
+                var credential = new ClientSecretCredential(
+                    Environment.GetEnvironmentVariable(aadTenantVar), 
+                    Environment.GetEnvironmentVariable(aadAppIdVar), 
+                    Environment.GetEnvironmentVariable(aadAppSecretVar));
```
Comment (benbp) [inline tools/notification-configuration/github-codeowner-subscriber/Program.cs:61]: Did these calls just not work before?

**Comment by benbp** [comment]
LGTM (sorry I didn't realize this was a draft until after reviewing).

File: tools/notification-configuration/github-codeowner-subscriber/Program.cs (hunk 1 above)
Comment (sima-zhu) [reply tools/notification-configuration/github-codeowner-subscriber/Program.cs:61]: This is what wes suggested. Env var is much safer than passing secret value.
He suggested we converted to env var

**Comment by sima-zhu** [comment]
> LGTM (sorry I didn't realize this was a draft until after reviewing).

I haven't solved build issue yet. Having the PR for easy discussion and testing on the air.

**Review by chidozieononiwu — APPROVED** [review]

**Pull request merged**